### DISCUSSION TITLE
Can I add advertisements to the IPTV list?

### DISCUSSION THREAD

**Discussion by wangjiming001** [iptv-org/community#501] [Q&A]
Can I add advertisements to the IPTV list?

**Comment by freearhey** [comment]
At the moment there is no such option, but if the @iptv-org/core decides to support this idea we can add it.

**Reply by PopeyeTheSai10r** [reply]
As long as it isn't intrusive advertising, I would consider this.

**Reply by BellezaEmporium** [reply]
I mean, if that's "infomercials-type" channels, why not. Like any channel, this still stumbles upon the category of a "TV Channel".

However, if it's for advertising a personal service or product... feels rather awkward to use our playlists to self-promote.

**Reply by wangjiming001** [reply]
This is just an advertisement to sell something. Sir, how can I contact you?

**Comment by Alstruit** [comment]
We have no control over the streams provided but would it not be fair to at least provide a disclaimer?
